src/components/portal/portal.tsx

```diff
@@ -236,7 +236,6 @@ export class Portal {
             parent.appendChild(element);
         });
 
-        this.hideContainer();
```

Comment:
Hmm, why are we not hiding it any more when removing the container?

Reply:
I wrote some text as commit message about why. But here is how I understand it. Does it make sense?
---

What is the purpose of `removeContainer()`?
1. It detaches the elements inside the portal from the container and reattaches them to their original parents.
2. It removes the portal container itself from the DOM.

I felt it doesn't make sense to hide the container in `removeContainer()`. If we’re removing the container from the DOM, the visibility (IS_VISIBLE_CLASS) of the container no longer matters. Once removed from the DOM, the `hideContainer()` method doesn’t need to run.

Am I wrong?

Reply:
It makes sense 👍